### PR TITLE
Add capturing bypass

### PR DESCRIPTION
**Pull request by plexus**
When using the output-capturing plugin, it is sometimes desirable to bypass it and still print things directly. 

See https://clojurians.slack.com/archives/CCY2V0U6A/p1670084873985659 for a workaround.

This applies the same logic, but provides a more generic `bypass` macro.

It seems `FileDescriptor` is not available in babashka, so until that changes this will be Clojure only.

### CODE REVIEW

**Comment by alysbrooks** [comment]
Yeah, I had a conversation with Michiel Borkent when there was an earlier issue with `FileDescriptor`, and he said `FileDescriptor` probably wasn't a good candidate for addition to Babashka.

**Review by alysbrooks — COMMENTED** [review]
Based on the Slack conversation, it sounds like this might be intended for using in test code by users of Kaocha? In that case, I think we'd ideally have documentation for it.

File: src/kaocha/testable.clj [hunk 1]
```diff
@@ -12,7 +12,7 @@
             [kaocha.result :as result]
             [kaocha.specs :refer [assert-spec]]
             [kaocha.util :as util])
-  (:import [clojure.lang Compiler$CompilerException]))
+  (:import (clojure.lang Compiler$CompilerException)))
```
[the remark this answers is not in the document]
Comment (alysbrooks) [reply src/kaocha/testable.clj:15]: To be clear, not arguing against this change, just wondering.

[the remark this answers is not in the document]
Comment (plexus) [reply src/kaocha/testable.clj:15]: The reason is that the first item in the list is special. In this case I agree with Alessandra Sierra https://stuartsierra.com/2016/clojure-how-to-ns.html

The practical result is that you get more sensible indentation

```clj
(:import (foo.bar Klass
                  OtherKlass))

(:import [foo.bar Klass
          OtherKlass])
```

**Comment by plexus** [comment]
Released in [v1.75.1190](https://github.com/lambdaisland/kaocha/releases/tag/v1.75.1190)

```clj
[lambdaisland/kaocha "1.75.1190"]                 ;; deps.edn
{lambdaisland/kaocha {:mvn/version "1.75.1190"}}  ;; project.clj
```

**Pull request merged**